src/components/FormQuestion.js

```diff
@@ -4,14 +4,17 @@ import { View, Text } from 'react-native';
 import CheckboxGroup from './CheckboxGroup';
 import RadioGroup from './RadioGroup.js';
 import DropDown from './DropDown';
+import TextField from './Login/TextFields.js';
+// import SectionHeader from './SectionHeader.js';
 // Utils
 import * as options from '../utilities/answerOptions.js';
 
 const RENDER_TYPES = {
   CHECKBOX: 'checkbox',
   DROPDOWN: 'dropdown',
   INPUT: 'input',
-  RADIO: 'radio'
+  RADIO: 'radio',
+  TITLE: 'sectionTitle'
```

Comment:
Render types are only for rendering answerOptions.  So sectionTitle would have to be a separate component.